tests/src/test/scala/whisk/core/entity/test/ExecHelpers.scala

```diff
@@ -13,21 +13,37 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
 package whisk.core.entity.test
 
+import org.scalatest.Matchers
+import org.scalatest.Suite
+
+import common.StreamLogging
+import common.WskActorSystem
+import whisk.core.WhiskConfig
 import whisk.core.entity._
 import whisk.core.entity.ArgNormalizer.trim
+import whisk.core.entity.ExecManifest._
+
+trait ExecHelpers
+    extends Matchers
+    with WskActorSystem
+    with StreamLogging {
+    self: Suite =>
+
+    private val config = new WhiskConfig(ExecManifest.requiredProperties)
+    ExecManifest.initialize(config) shouldBe true
 
-trait ExecHelpers {
     protected val NODEJS = "nodejs"
     protected val NODEJS6 = "nodejs:6"
     protected val SWIFT = "swift"
     protected val SWIFT3 = "swift:3"
 
-    protected def js(code: String, main: Option[String] = None): Exec = CodeExecAsString(NODEJS, trim(code), main.map(_.trim), false)
-    protected def js6(code: String, main: Option[String] = None): Exec = CodeExecAsString(NODEJS6, trim(code), main.map(_.trim), false)
-    protected def swift(code: String, main: Option[String] = None): Exec = CodeExecAsString(SWIFT, trim(code), main.map(_.trim), true)
-    protected def swift3(code: String, main: Option[String] = None): Exec = CodeExecAsString(SWIFT3, trim(code), main.map(_.trim), false)
+    protected def js(code: String, main: Option[String] = None): Exec = CodeExecAsString(RuntimeManifest(NODEJS), trim(code), main.map(_.trim))
+    protected def js6(code: String, main: Option[String] = None): Exec = CodeExecAsString(RuntimeManifest(NODEJS6), trim(code), main.map(_.trim))
+    protected def swift(code: String, main: Option[String] = None): Exec = CodeExecAsString(RuntimeManifest(SWIFT, deprecated = Some(true)), trim(code), main.map(_.trim))
```

Comment:
If this refers to the Swift 2 runtime, then is it still needed here? It also appears in the [manifest deployment](https://github.com/openwhisk/openwhisk/blob/50aa4ee7a194b7b25d029cf827cb0ea00f5c1f2a/ansible/group_vars/all#L19), which seems confusing.